packages/components/src/ecosystem_bar/index.stories.jsx

```diff
@@ -1,4 +1,4 @@
-import EcosystemBar from '@/src/ecosystem_bar/index';
+import EcosystemBar from '.';
```

Comment:
this seems to not work in the monorepo type of setup..

Reply:
Interesting... Does 
`import EcosystemBar from './index'; `
work for you? That would be the usual way of importing, maybe we could add it here to keep it consistent?